lib/services/ios-project-service.ts

```diff
@@ -186,7 +186,7 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 	/**
 	 * Exports .xcarchive for AppStore distribution.
 	 */
-	public async exportArchive(projectData: IProjectData, options: { archivePath: string, exportDir?: string, teamID?: string }): Promise<string> {
+	public async exportArchive(projectData: IProjectData, options: { archivePath: string, exportDir?: string, teamID?: string, exportMethod?: string, xcargs?: string }): Promise<string> {
```

Comment:
@Plamen5kov @rosen-vladimirov Would I change it here?  https://github.com/NativeScript/nativescript-cli/blob/00b80f71f6e45955c09b091adc884e74e41aca47/lib/commands/appstore-upload.ts

The filename is appstore-upload so I'm not sure if this would be where to change it, since I'm not uploading to the app store